tron/core/jobrun.py

```diff
@@ -207,12 +207,24 @@ def _start_action_runs(self):
         """Start all startable action runs, and return any that were
         successfully started.
         """
-        return [
+        started_runs = [
             action_run
             for action_run in self.action_runs.get_startable_action_runs()
             if action_run.start()
         ]
 
+        if not started_runs:
+            blocked_on_triggers = [
+                r
+                for r in self.action_runs
+                if r.is_blocked_on_trigger
```

Comment:
there is an inconsistency between this and line 223 - is this supposed to be triggers or trigger?